README.md

```diff
@@ -28,9 +28,10 @@ If you're interested in learning more about our governance or becoming a member
 ## Join the fun!
 
 * We have a [Google Hangouts](https://hangouts.google.com/call/BgkpIXkZghZH92NqjNWsAEEI) meeting every other Wednesday at **18:00 UTC**.
-* [This link](https://www.google.com/search?q=1800+utc+in+local+time) should convert that time to your local time zone's time.
-* If you'd like to use Google Calendar to track meeting days you can follow this event: <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWgxZnN2cWR2NjBhZGlhODBwNXRoc3RrMDBfMjAxOTA3MTBUMTkwMDAwWiAxM3JhM3R1ZXFrcWJmZnBmMmc1NmZvMmN0c0Bn&amp;tmsrc=13ra3tueqkqbffpf2g56fo2cts%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>    
+  * [This link](https://www.google.com/search?q=1800+utc+in+local+time) should convert that time to your local time zone's time.
+  * If you'd like to use Google Calendar to track meeting days you can follow this event: <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=MWgxZnN2cWR2NjBhZGlhODBwNXRoc3RrMDBfMjAxOTA3MTBUMTkwMDAwWiAxM3JhM3R1ZXFrcWJmZnBmMmc1NmZvMmN0c0Bn&amp;tmsrc=13ra3tueqkqbffpf2g56fo2cts%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>    
 * We have a `wg-gamedev` channel in the [Offical Rust Discord][discord].
+* We have a `Rust Gamedev WG` room on [Matrix](https://matrix.to/#/!MljVigmyqZqNMImsdX:matrix.org)
```

Comment:
The address should be `https://matrix.to/#/#rust-gamedev-wg:matrix.org`

Reply:
Ah, I think the "share" link just gave me the hashed version.